fulp_modules/ipc/code/surgery/chassis_restoration.dm

```diff
@@ -0,0 +1,77 @@
+/datum/surgery/robot_chassis_restoration
+	name = "Full chassis restoration (Full heal + revival)"
```

Comment:
restoration should be enough for people to realize it revives, no need to make it weirdly ocky